tools/build-amber-sample.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# Copyright 2019 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ $1 == "" ]]; then
+  echo "Usage: $0 [build directory]"
+  exit 1
+fi
+
+BUILD_DIR=$(readlink -f $1)
+if [[ $(ls $BUILD_DIR 2> /dev/null) == "" ]]; then
+  mkdir -p $BUILD_DIR
+fi
+
+if [[ $ANDROID_SDK_HOME == "" ]]; then
+  echo "Error: ANDROID_SDK_HOME missing, please set env variable e.g.,"
+  echo "       $ export ANDROID_SDK_HOME=path/to/Android/SDK"
+  exit 1
+fi
+
+if [[ $ANDROID_NDK_HOME == "" ]]; then
+  echo "Error: ANDROID_NDK_HOME missing, please set env variable e.g.,"
+  echo "       $ export ANDROID_NDK_HOME=path/to/Android/NDK"
+  exit 1
+fi
+
+if [[ $(command -v javac) == "" ]]; then
+  echo "Error: Install Java. Recommended version is Java 8."
+  exit 1
+fi
+
+if [[ $KEY_STORE_PATH == "" ]]; then
+  echo "Error: KEY_STORE_PATH missing, please set env variable."
+  exit 1
+fi
+
+set -x
+
+ANDROID_SOURCE_DIR=$(dirname $(readlink -f $0))/../android_sample
+
+APK_NAME=AmberSample.apk
+ANDROID_PLATFORM=android-28
+ANDROID_BUILD_TOOL_VERSION=28.0.0
+ABI=arm64-v8a
+BUILD_TYPE=Debug
```

Comment:
Does building Debug vs Release matter?

Reply:
No. Debug might be slow and changed it as Release.